spec/fixtures/controllers/supplementals/index.json

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
@chrisdolendo how did you generate this file? We need to update it but I'm not sure how you made it in the first place